### PR TITLE
Display randomized art image and details on page load

### PR DESCRIPTION
**Pull request by nbalmoori**
# Description
This PR will:

- Refactor API fetch call and component structure to have initial state be set to a randomized art object
- Display an art image and details on page load

## Issues

Closes #

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [ ] Cosmetic changes (changes to layout or visuals in CSS file)

## How Has This Been Tested?

- [x] console.log()
- [x] dev tools
- [x] functionality check through web browser
- [ ] tested by viewing the markdown file preview
- [ ] tested with testing framework (Cypress)

## Checklist:

- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have deleted all comments/console.log() after functionality and understanding is final
- [x] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers

### CODE REVIEW

File: src/features/currentArt/currentArt.js [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-//   const currentState = useSelector((state) => state)
-//   const dispatch = useDispatch()
-//   return (
-//     <div onClick={() => dispatch(randomize(currentState, fetchData))}>
-//       <img src=''/>
-//     </div>
-//   )
-// }
-// export default Art
+import React from 'react'
```
Comment (whiteheadol) [inline src/features/currentArt/currentArt.js:1]: I'm so glad you figured this out, Nikki!

File: src/features/currentArt/currentArtSlice.js [hunk 2]
```diff
@@ -1,32 +1,55 @@
...
-
-// const fetchArt = createAsyncThunk(
-//   'currentArt/randomize',
-//   async (arg, thunkAPI) => {
-//     const response = await fetchData()
-//     return response.data
-//   }
-// )
 
+export const getCurrentArt = createAsyncThunk('currentArt/getCurrentArt', async () => {
```
Comment (Jlavarine) [inline src/features/currentArt/currentArtSlice.js:3]: What a great use of asyncThunk

File: src/features/currentArt/currentArtSlice.js [hunk 3]
```diff
@@ -1,32 +1,55 @@
...
+    return {
+      title: data.title,
+      artist: artist || 'unknown',
+      medium: data.medium,
+      century: data.century,
+      image: data.images[0].baseimageurl
+    }
+  }
+
+  return fetch(`https://api.harvardartmuseums.org/object/${id}/?apikey=b5915d6a-dcba-45df-a4a6-9ff3a72dfbeb`)
```
Comment (Jlavarine) [inline src/features/currentArt/currentArtSlice.js:24]: So glad it can be dynamic!

**Review by Jlavarine — APPROVED** [review]
Great job!

**Pull request merged**